binding.gyp

```diff
@@ -1,8 +1,18 @@
 {
-	"targets": [ {
-		"target_name":  "journal_send",
-		"sources":      [ "src/journal_send.cc" ],
-		"include_dirs": [ "<!(node -e \"require('nan')\")" ],
-		"libraries":    [ "<!@(pkg-config --libs-only-l --silence-errors libsystemd || pkg-config --libs-only-l libsystemd-journal)" ]
-	} ]
+	"targets": [
+		{
+			"target_name": "journal_send",
+			"sources": [
+				"src/journal_send.cc"
+			],
+			"libraries": [
+        "-lsystemd",
+      ],
+			'cflags!': [ '-fno-exceptions' ],
+      'cflags_cc!': [ '-fno-exceptions' ],
```

Comment:
What's the reason for the appended `!`?

Reply:
Those are flags required for disabling exception support. So all the exceptions are bubbled up to V8 engine. They are pretty standard in NAPI modules. It is very hard to compile applications if those two flags are not removed from configuration. 

<img width="1027" alt="image" src="https://user-images.githubusercontent.com/5595263/127320662-25aaa771-a1d7-4af5-82c4-ce0e7c147c0f.png">

Reply:
to be more accurate _**Compiling with `fno-exceptions` disables exceptions support and uses the variant of C++ libraries without exceptions. Use of try, catch, or throw results in an error message.**_

Reply:
Yes, I understand its purpose. I was just wondering about `'cflags'` vs `'cflags!'`? I haven't found a proper description with the help of a Google search. I suspect, it can override cflags, if they are defined else-where, like in the target section.

Reply:
correct, That's why I included the image in the comment above

Reply:
Sorry for the dumb questions. But if it removes the `-fno-exceptions` flag, the code will be compiled without the `-fno-exceptions` flag? And you want exceptions to be removed, right?

Reply:
Left: your version. Right: `'cflags'` and `'cflags_cc'` instead of `'cflags!'` and `'cflags_cc!'`

![Screenshot from 2021-07-28 17-29-24](https://user-images.githubusercontent.com/6105784/127350904-e18124e3-8c27-4404-acba-92847e774189.png)

Reply:
I think you are right. it is like negative of negative.. You want me to remove those?

Reply:
I made a suggestion picking up what we discovered ;)